src/WheelsAndGoods.DataAccess.Migrations/Migrations/20220818165257_AddRefreshTokenTable.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace WheelsAndGoods.DataAccess.Migrations.Migrations
+{
+    public partial class AddRefreshTokenTable : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "refresh_token",
+                columns: table => new
+                {
+                    id = table.Column<Guid>(type: "uuid", nullable: false),
+                    access_token_id = table.Column<Guid>(type: "uuid", nullable: false),
+                    created_at_utc = table.Column<DateTime>(type: "timestamp without time zone", nullable: false),
+                    is_invalidated = table.Column<bool>(type: "boolean", nullable: false),
+                    user_id = table.Column<Guid>(type: "uuid", nullable: false)
+                },
+                constraints: table =>
```

Comment:
FK is absent. You should add FK in entity configuration and regenerate migration